nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -714,6 +703,32 @@ Some processors have properties that refer to other components, such as Controll
 
 image:create-service-ssl-context.png["Create Service"]
 
+==== Relationships Tab
+
+The Relationships tab contains an 'Automatically Terminate / Retry Relationships' section. Each of the Relationships that is defined by the Processor is listed here, along with its description.
+
+image::relationships-tab.png["Relationships Tab"]
+
+===== Automatically Terminate
+In order for a Processor to be considered valid and able to run, each Relationship defined by the Processor must be either connected to a downstream component or auto-terminated. If a Relationship is auto-terminated, any FlowFile that is routed to that Relationship will be removed from the flow and its processing considered complete. Any Relationship that is already connected to a downstream component cannot be auto-terminated. The Relationship must first be removed from any Connection that uses it. Additionally, for any Relationship that is selected to be auto-terminated, the auto-termination status will be cleared (turned off) if the Relationship is added to a Connection.
+
+===== Automatically Retry
+Users can also configure whether or not FlowFiles routed to a given Relationship should be retried.
```

Comment:
It would probably be a good idea to mention that if both retry and terminate are selected for a given relationship, retry attempts will happen first, then termination.

Reply:
Disregard. I just noticed your note at line 730.